packages/core/src/connectivity/scp-cf/jwt.ts

```diff
@@ -21,27 +21,23 @@ const logger = createLogger({
  * @param token - JWT to be decoded
  * @returns Decoded payload.
  */
-export function decodeJwt(token: string): DecodedJWT {
+export function decodeJwt(token: string): JwtPayload {
   return decodeJwtComplete(token).payload;
 }
 
 /**
  * Decode JWT and return the complete decoded token.
- * @param token - JWT to be decoded
+ * @param token - JWT to be decoded.
  * @returns Decoded token containing payload, header and signature.
  */
-export function decodeJwtComplete(token: string): CompleteDecodedJWT {
-  const decodedToken = jwt.decode(token, { complete: true });
+export function decodeJwtComplete(token: string): Jwt {
```

Comment:
Same question as above are `Jwt` and `CompleteDecodedJWT` type compatible?

Reply:
Yes, https://github.com/SAP/cloud-sdk-js/pull/1368#discussion_r656025349